README.md

```diff
@@ -27,19 +27,15 @@ Building
 
 To build relx and generate a standalone escript executable:
 
-    $ make
+    $ ./rebar3 escriptize
```

Comment:
I assume this automatically runs all tests? Also, we may need to change travis once this is in, or does it recognize enough to know it is an erlang project and tries to run rebar3 first?